src/screens/faq/styles.tsx

```diff
@@ -0,0 +1,100 @@
+import { makeStyles } from '@material-ui/styles';
+import { useTheme } from '@material-ui/core/styles';
+import { getMinMediaQuery } from '@styles';
+
+export const useGetStyles = () => {
+  const theme: any = useTheme();
+  const useStyles = makeStyles({
+    root: {
+      padding: '1rem 1rem 2.5rem',
+    },
+    title: {
+      fontSize: '1.375rem',
+      fontWeight: 500,
+      margin: '0',
+      marginBottom: '0.5rem',
+      marginTop: '1rem',
+      lineHeight: '1.375rem',
+      color: '#000000',
```

Comment:
You should really take a look at how theme works. I recently updated the `themes.js` to follow our designer's colors and handle darkmode / lightmode color switch

Reply:
done